app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -208,6 +214,14 @@ public static void urlBarEvent(boolean isUrl) {
         }
     }
 
+    private static TelemetryEvent isDefaultBrowserEvent() {
+        final Context context = TelemetryHolder.get().getConfiguration().getContext();
+        final Browsers browsers = new Browsers(context, "http://www.mozilla.org");
+        final String isDefault = browsers.isDefaultBrowser(context) ? Value.TRUE : Value.FALSE;
+
+        return TelemetryEvent.create(Category.PREF, Method.BACKGROUND, Object.IS_DEFAULT, isDefault);
```

Comment:
My only concern here is: How do we analyze this on the server side? What we'd want to see in re:dash is a pie chart showing the % of using Focus as a default browser. Can we do this easily with an event?

If there wouldn't be a code freeze would we use something else? Like a field in the event ping for that? In this case I'd prefer to move this to the next iteration. What do you think?